components/card/src/vwc-card.ts

```diff
@@ -63,37 +65,42 @@ export class VWCCard extends LitElement {
 	}
 
 	protected override render(): unknown {
-		const actionsClassMap = {
-			'no-content': !(this.shouldShowActionsSlot)
+		const footerClassMap = {
+			'no-content': !(this.#shouldShowFooterSlot)
 		};
 		return html`
 			<div class="vwc-card">
 				<div class="vwc-card-media">
 					<slot name="media"></slot>
 				</div>
-				<div class="vwc-card-info">
-					${this.renderHeader()}
-					<div class="vwc-card-supportText">
-							${this.supportingText ? this.supportingText : ''}
-					</div>
-					<div class="vwc-card-actions ${classMap(actionsClassMap)}">
-							<slot name="actions" @slotchange="${this.actionsSlotChanged}"></slot>
-					</div>
+				<div class="vwc-card-content">
+					<slot name="content">
+						${this.renderHeader()}
+						<div class="vwc-card-carText">
+							${this.carText ? this.carText : nothing}
+						</div>
+					</slot>
+				</div>
+				<div class="vwc-card-footer ${classMap(footerClassMap)}">
+					<slot name="footer" @slotchange="${this.footerSlotChanged}"></slot>
 				</div>
 			</div>
 		`;
 	}
 
 	private renderHeader() {
 		return html`
-			<header class="${this.headerClass}">
-				<div class="vwc-card-header">
-					<slot name="graphics" @slotchange="${this.graphicsSlotChanged}">
-						${this.headerIcon ? this.renderIcon() : ''}
-					</slot>
-					<div class="vwc-card-title">${this.heading}</div>
+			<header class="vwc-card-header ${this.headerClass}">
+				<div class="vwc-card-header-content">
+						<slot name="graphics" @slotchange="${this.graphicsSlotChanged}">
+							${this.headerIcon ? this.renderIcon() : ''}
```

Comment:
`headerIcon` is also too long, there's no other icon, so why plainly `icon`. I know it was here before but maybe make the interface simpler?

Reply:
changed to _icon_